### PR TITLE
Theme tweaks

### PR DESCRIPTION
**Pull request by rvkutd**
considerable changes in the theme from Reon  ( the bbs theme )

### CODE REVIEW

File: themes/bbs/templates/node/node--ding-group.tpl.php [hunk 1]
```diff
@@ -77,12 +77,11 @@
  * @see template_preprocess_node()
  * @see template_process()
  */
-$img_url = $node->field_ding_group_title_image['und'][0]['uri'];
 ?>
 <div class="<?php print $classes; ?>"<?php print $attributes; ?>>
   <div class="bbs-top-page-content">
     <div class="bbs-page-banner-image">
-      <img class="banner-image" src="<?php print file_create_url($img_url); ?>"/>
+      <?php print render($content['field_ding_group_title_image']); ?>
```
Comment (xendk) [inline themes/bbs/templates/node/node--ding-group.tpl.php]: :+1:

File: themes/bbs/template.php [hunk 2]
```diff
@@ -171,7 +171,7 @@ function bbs_menu_link__menu_bbs_topbar_menu($vars) {
         default:
             $element['#localized_options']['attributes']['class'][] = 'topbar-link-menu';
             $element['#attributes']['class'][] = 'topbar-link-menu';
-            $output = '<div class="topbar-link-menu-inner">' . '<span></span><span></span><span></span><span></span>' . '</div>';
+            $output = '<div class="topbar-link-menu-inner">' . '<span></span><span></span><span></span><span></span>' . '<div class="details">' . $element['#title'] . '</div>' . '</div>';
```
Comment (xendk) [inline themes/bbs/template.php]: Span, span, span, span, span, eggs and span?

Would be nice with a comment as to why these spans are needed...

**Comment by tripox** [comment]
screenshot 2019-01-14 at 17 31 51
screenshot 2019-01-14 at 17 31 44

Looks like new errors was introduced too.

**Comment by tripox** [comment]
screenshot 2019-01-15 at 14 44 30

**Pull request merged**